### ISSUE TITLE
Input best bets for guidance search as provided by OGC

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3704]
### Summary

**What we are after:**
_In our conversations with OGC, they've agreed to provide a list of "best bets" to aid the guidance search to find some specific terms._

### Related issues

#3671 - Modifications based on OGC's testing comments

### Action items: 

- [x] Send OGC a screenshot of what's needed to create a best bet and ask them to provide the list.
- [x] Create the best bets once OGC provides them.

### Completion criteria

- [ ] Best bets provided by OGC added to search.gov's list of best bets for the s3 bucket.

**Comment by dorothyyeager** [comment]
Have sent the screencap and description of fields needed to create a best bets search to OGC. Moving to blocked until we hear back with the terms they'd like a best bets search for.

**Comment by dorothyyeager** [comment]
OGC requested best bets be added for each court case press release. I've added that. Some interesting notes:

1. Adding them produces duplicate results as search.gov finds the page as normal, but also finds the best bet
2. Adding the best bet allows us to customize the description which is nice. 
3. Adding a best bet makes the text description non-italicized. 

We'll want to take a look in a few weeks to review how well they are working and if we want to add them for Forms or Guides. I'll put in a follow-up ticket for a future sprint. cc: @AmyKort @patphongs @JonellaCulmer @kathycarothers 

For now, we've added the OGC-requested best bets so closing this ticket.
